src/math_tools/curl.hpp

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include "ddc_alias_inline_functions.hpp"
+#include "ddc_aliases.hpp"
+#include "gradient.hpp"
+#include "metric_tensor_evaluator.hpp"
+#include "static_tensors.hpp"
+#include "vector_field.hpp"
+
+/**
+ * @brief A class which implements a curl operator
+ * The implemented equation is:
+ * @f$ \nabla \cross \mathbf{F} @f$
+ * @f$ \nabla \cross \mathbf{F} = {\cal J}_{\rm x}^{-1}\epsilon^{klm}\partial_{x^l} F_m @f$
```

Comment:
```suggestion
 * @f$ \nabla \cross \mathbf{F} = {\cal J}_{\rm x}^{-1}\epsilon^{klm}\partial_{x^m} F_l @f$
```

Reply:
Never mind sorry. I've just checked this this morning and I am wrong. The docstring is fine.